src/managed/OpenLiveWriter.PostEditor/PostEditorMainControl.cs

```diff
@@ -1645,11 +1641,12 @@ public void SaveRibbonSettings()
 
         private delegate int WithIStreamAction(IStream stream);
         /// <summary>
-        ///
+        /// Withes the ribbon settings i stream.
         /// </summary>
         /// <param name="create">Create the stream if it doesn't exist, and make it writable.</param>
-        /// <param name="onlyIfChanged"></param>
-        /// <param name="action"></param>
+        /// <param name="writable">if set to <c>true</c> [writable].</param>
+        /// <param name="onlyIfChanged">if set to <c>true</c> [only if changed].</param>
+        /// <param name="action">The action.</param>
```

Comment:
Sort of, I used GhostDoc as one of them was missing (writable one). I can remove if preferred.